news/2020-05-08-mappers-report-april.md

```diff
@@ -0,0 +1,193 @@
+
+If you have any topics which should be discussed in future meetings, or a proposal you'd like to push to NAT, please do not hesitate to make a post about it on the [NAT Meetings Overview](https://osu.ppy.sh/community/forums/topics/1041618)! 
+
+Once enough topics are gathered, future NAT meeting dates will be announced in further Mappers' Report issues, in #modding on the [osu!dev Discord server](https://discord.gg/ppy) and on the forums!
+
+### [April 11th, 2020 - Genre and Language Settings](https://osu.ppy.sh/community/forums/topics/1059659)
+
+Recent changes to the website allow us to set genre and language once more, a feature which had only been available on the old website for a long time. With these changes, we decided to revisit what genres and languages are available and work out some of the kinks in the current list. 
+
+Quite a few changes have been proposed like adding popular genres such as Metal; Classical; Folk and adding new categorizations like Vocaloid or Doujin. You can find the full list of suggested changes [on github](https://github.com/ppy/osu-web/issues/5852#issuecomment-616219839), where discussion is still taking place.
+
+No additional meetings have been planned so far, but we'll be sure to get the word out when there's new topics on the horizon!
+
+## <a id="within-bn-nat"></a>Within the Beatmap Nominators and NAT
+
+New Beatmap Nominators! Quick, request all your beatmaps! Jokes aside, the Beatmap Nominators are never resting. New people come, old people go. Here you can see what happened within the Beatmap Nominators this month!
+
+### BN Additions
+
+- **osu!:** [Agatsu](https://osu.ppy.sh/users/5579871)
+- **osu!taiko:** [Greenshell](https://osu.ppy.sh/users/8693851)
+- **osu!catch:** [Dako](https://osu.ppy.sh/users/11081858)
+- **osu!mania:** [Sun](https://osu.ppy.sh/users/4115819)
+
+Ever been interested in getting a Beatmap Nominator and deciding what maps get ranked next? Or you just want to help new mappers to get their beatmaps ranked? Then head right over to the [Beatmap Nominator website](https://bn.mappersguild.com/) and apply today! 
+
+### BN Resignations
+
+- **osu!:** [celerih](https://osu.ppy.sh/users/4696296), [Chinochan](https://osu.ppy.sh/users/3626063), [Uta](https://osu.ppy.sh/users/9439796) & [Ryuusei Aika](https://osu.ppy.sh/users/7777875)
+- **osu!catch:** [Daletto](https://osu.ppy.sh/users/7592136) & [Lacrima](https://osu.ppy.sh/users/4915649)
+
+### NAT Additions
+
+Our office cat **[Mao](https://osu.ppy.sh/users/2204515)** has returned! After stepping down from the NAT duties in December 2019, he decided to make his return. Mao will lend the other osu! NATs a helping hand with doing Beatmap Nominator evaluations. Welcome back!
+
+## <a id="mappers-showcase"></a>Mappers' Showcase
+
+The Mappers' Showcase will bring to you Mappers from every game mode which not everyone might know. They are hand-selected and a hidden gem for their game mode. 
```

Comment:
this capitalisation feels a little... German? It's not a given name so lowercase is probably fine.

Second sentence could maybe use a rewording (sounds a little objectifying I guess? like it's talking about a map, and not a mapper) but don't have a concrete suggestion just yet.
```suggestion
The Mappers' Showcase will bring to you mappers from every game mode which not everyone might know. They are hand-selected and a hidden gem for their game mode. 
```